sui_types/src/crypto.rs

```diff
@@ -393,6 +393,19 @@ impl AuthoritySignature {
     }
 }
 
+pub trait AuthoritySignInfoTrait: Eq + Serialize + serde::de::DeserializeOwned {}
```

Comment:
It is uncommon to need an alias for a handful of very common traits, why not use the trait bounds directly?

Is it because you have some trouble with the lifetimes in the `Deserialize` trait? I found [this page](https://serde.rs/lifetimes.html#trait-bounds) helpful to get the hang of that.